### PR TITLE
Load phones from api

### PR DESCRIPTION
**Pull request by Helen-arch**
Fetching phones data
I will customize selects in my next PR
Screenshot 2023-12-13 at 16 40 56

### CODE REVIEW

File: src/modules/shared/components/ProductsList/ProductsList.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import { Phone } from '../../../../types/Phone';
+
+type Props = {
+  phones: Phone[];
+};
+
+export const ProductsList: React.FC<Props> = ({ phones }) => (
+  <div className={styles.itemsContainer}>
+    {phones.map(({
+      // itemId,
```
Comment (Clover247) [inline src/modules/shared/components/ProductsList/ProductsList.tsx:12]: here

File: src/modules/shared/components/Card/Card.tsx [hunk 2]
```diff
@@ -16,7 +16,7 @@ type Props = {
 };
 
 export const Card: React.FC<Props> = ({
-  itemId,
+  // itemId,
```
Comment (Clover247) [inline src/modules/shared/components/Card/Card.tsx:19]: here

File: src/modules/shared/components/ProductsList/ProductsList.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      name,
+      price,
+      fullPrice,
+      screen,
+      capacity,
+      ram,
+    }) => {
+      return (
+        <Card
+          // itemId={itemId}
```
Comment (Clover247) [inline src/modules/shared/components/ProductsList/ProductsList.tsx:23]: here

**Review by Clover247 — APPROVED** [review]
Amazing job! I'm not sure about the commented "itemId", if you don't need them, can you delete them?

**Review by Maksim114488 — APPROVED** [review]
Nice Job

File: src/modules/shared/components/ProductsList/ProductsList.tsx (hunk 1 above)
Comment (Helen-arch) [reply src/modules/shared/components/ProductsList/ProductsList.tsx:12]: we will need them for handlers later

**Pull request merged**